### PR TITLE
banner updates

### PR DESCRIPTION
**Pull request by epwinchell**
Jira: https://issues.redhat.com/browse/RHCLOUD-24603

- Change dots to dashes / update colors
- shorten slides
- update padding around first panel and carousel

Old
Screenshot 2023-04-04 at 1 23 24 PM

New
Screenshot 2023-04-04 at 1 21 47 PM

### CODE REVIEW

File: src/components/app-content-renderer/SimpleSlider.js [hunk 1]
```diff
@@ -59,30 +58,27 @@ function SimpleSlider() {
...
-                </Text>
-                <Flex className="pf-u-flex-direction-row-on-md">
-                  <Text component="p" className="pf-u-pr-sm pf-u-mb-0">
-                    <Link to="/openshift/sandbox">
-                      <Button variant="danger" className="pf-u-pr-sm">
-                        Build apps on the sandbox
-                      </Button>
-                    </Link>
+                  configuration for free with your Red Hat Developer
+                  account.&nbsp; &nbsp;
```
Comment (karelhala) [inline src/components/app-content-renderer/SimpleSlider.js:62]: Just out of curiosity, why two non breaking spaces? Wouldn't it be better to use `&emsp` or regular `padding-left` with PF's gutter?

File: src/components/app-content-renderer/SimpleSlider.js [hunk 2]
```diff
@@ -59,30 +58,27 @@ function SimpleSlider() {
...
-                  <Text component="a">
-                    <Button
-                      href="https://developers.redhat.com/about"
-                      variant="tertiary"
-                      className="land-c-button-custom"
-                    >
-                      Learn about the program
+                </Text>
+                <Text component="p" className="pf-u-pr-sm pf-u-mb-0">
+                  <Link to="/openshift/sandbox">
```
Comment (karelhala) [inline src/components/app-content-renderer/SimpleSlider.js:72]: This doesn't follow the `beta`/`preview` either use `openshift/sandbox` (which will just append the text) or `./openshift/sandbox` which will use baseURI parameter. Or import `useChrome` and call `appNavClick`

**Review by karelhala — COMMENTED** [review]
Looking good, just two minor enhancements and we are golden!

File: src/components/app-content-renderer/SimpleSlider.js (hunk 2 above)
Comment (epwinchell) [reply src/components/app-content-renderer/SimpleSlider.js:72]: @karelhala Every link on the Landing page is set up that same way. Do they all need to be changed? If so, can that be done separately?

Comment (karelhala) [reply src/components/app-content-renderer/SimpleSlider.js:71]: Yeah, might be better to go trough the routes and fix them properly to react-router v6.

**Review by karelhala — APPROVED** [review]

**Pull request merged**